docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -34,43 +34,47 @@ By default, columns are ordered according to the order they are included in the
-
-{{"demo": "pages/components/data-grid/columns/HeaderColumnsGrid.js", "bg": "inline"}}
 
-For more advanced header configuration, go to the [rendering section](/components/data-grid/rendering/#header-cell).
+{{"demo": "pages/components/data-grid/columns/HeaderColumnsGrid.js", "bg": "inline", "defaultCodeOpen": false}}
 
-## Column width
+### Width
 
-By default, the columns have a width of 100 pixels.
-This is an arbitrary, easy to remember value.
-To change the width of a column, use the `width` property available in `GridColDef`.
+The default columns width is 100 pixels.
+- set the `width` property available in `GridColDef` to override that value.
 
-{{"demo": "pages/components/data-grid/columns/ColumnWidthGrid.js", "bg": "inline"}}
+{{"demo": "pages/components/data-grid/columns/ColumnWidthGrid.js", "bg": "inline",  "defaultCodeOpen": false}}
 
-### Fluid width
+#### Fluid width
 
-Each column has a fixed width of 100 pixels by default, but column fluidity (responsiveness) can be by achieved by setting the `flex` property in `GridColDef`.
+Column fluidity or responsiveness can be by achieved by setting the `flex` property in `GridColDef`.
 
-The `flex` property accepts a value between 0 and ∞.
+- The `flex` property accepts a value between 0 and ∞.
+- it works by dividing the remaining space in the grid among all flex columns in proportion to their `flex` value.
 
-The `flex` property works by dividing the remaining space in the grid among all flex columns in proportion to their `flex` value.
 For example, consider a grid with a total width of 500px that has three columns: the first with `width: 200`; the second with `flex: 1`; and third with `flex: 0.5`.
 The first column will be 200px wide, leaving 300px remaining. The column with `flex: 1` is twice the size of `flex: 0.5`, which means that final sizes will be: 200px, 200px, 100px.
 
-Note that `flex` doesn't work together with `width`. If you set both `flex` and `width` in `GridColDef`, `flex` will override `width`.
-
-In addition, `flex` does not work if the combined width of the columns that have `width` is more than the width of the grid itself. If that is the case a scroll bar will be visible, and the columns that have `flex` will default back to their base value of 100px.
+**Note** 
+- `flex` doesn't work together with `width`. If you set both `flex` and `width` in `GridColDef`, `flex` will override `width`.
+- `flex` does not work if the combined width of the columns that have `width` is more than the width of the grid itself. If that is the case a scroll bar will be visible, and the columns that have `flex` will default back to their base value of 100px.
 
 {{"demo": "pages/components/data-grid/columns/ColumnFluidWidthGrid.js", "bg": "inline"}}
 
-## Column resizing [<span class="pro"></span>](https://material-ui.com/store/items/material-ui-pro/)
+### Hiding
+
+Set the column definiton attribute
+- `hide`: Hide the column.
+
+```tsx
+const hiddenColumn = {field: 'id', hide: true};
```

Comment:
It's not following prettier format, but more importantly, I would advocate is confusing. I think that 

```suggestion
<DataGrid columns={[{ field: 'id', hide: true }]} />
```

would be a lot clearer.